app/forms/support/school_form.rb

```diff
@@ -2,9 +2,8 @@
 
 module Support
   class SchoolForm < Form
-    MAIN_SITE = 'main site'
+    # MAIN_SITE = 'main site'
```

Comment:
Do we need this? I'd prefer we delete rather than comment out

Reply:
Yep